raiden/network/protocol.py

```diff
@@ -655,24 +656,46 @@ def receive(self, data):
             waitack = self.senthashes_to_states.get(message.echo)
 
             if waitack is None:
-                if log.isEnabledFor(logging.INFO):
-                    log.info(
+                if log.isEnabledFor(logging.DEBUG):
+                    log.debug(
                         'ACK FOR UNKNOWN ECHO',
                         node=pex(self.raiden.address),
-                        echohash=pex(message.echo)
+                        echohash=pex(message.echo),
                     )
 
             else:
-                if log.isEnabledFor(logging.INFO):
-                    log.info(
+                if log.isEnabledFor(logging.DEBUG):
+                    log.debug(
                         'ACK RECEIVED',
                         node=pex(self.raiden.address),
                         receiver=pex(waitack.receiver_address),
-                        echohash=pex(message.echo)
+                        echohash=pex(message.echo),
                     )
 
                 waitack.async_result.set(True)
 
+        elif isinstance(message, Ping):
+            if ping_log.isEnabledFor(logging.DEBUG):
+                ping_log.debug(
+                    'PING RECEIVED',
+                    node=pex(self.raiden.address),
+                    echohash=pex(echohash),
+                    message=message,
```

Comment:
can you also add a `sender=message.sender` here?

Reply:
done